_download-meta/discord-social-releases.md

```diff
@@ -0,0 +1,9 @@
+---
+---
+
+This plugin enables OpenTTD's integration with Discord. You need OpenTTD 14.0 or later.
+
+For Linux / Windows, to install the plugin, extract the download into your `social_integration` folder, under OpenTTD in your documents folder.
+For MacOS, simply run the `Install`.
+
+At any time you can remove the plugin from the `social_integration` folder to disable the integration.
```

Comment:
Pedant, and not sure whether these nuances should be in the documentation:
* removing the library while the game is running does not disable the integration at that moment.
* on Windows it might be that you are not allowed to remove the library while the game is running, as you cannot remove files that are in use.

Maybe something along the lines of the following helps?
```suggestion
At any time, when OpenTTD is not running, you can remove the plugin from the `social_integration` folder to disable the integration.
```

Reply:
Excellent suggestion, tnx :D